controllers/suite_test.go

```diff
@@ -39,10 +38,7 @@ var testEnv *envtest.Environment
 
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
-
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+	RunSpecs(t, "Controller Suite")
```

Comment:
For history: the envtest/printer was removed from the controller-runtime repo in https://github.com/kubernetes-sigs/controller-runtime/pull/1977

Following what others have done: https://github.com/csi-addons/kubernetes-csi-addons/commit/74c24b3674aedf0e7eeea427b6d1be171ab14eb9